### PR TITLE
v5.17.1

### PR DESCRIPTION
**Pull request by pierrecamilleri**
- fix: deprecated dependencies ([PR 1674](https://github.com/frictionlessdata/frictionless-py/pull/1674))
- fix: unexpected "missing-label" error with option `header_case = False` ([#1635](https://github.com/frictionlessdata/frictionless-py/issues/1635))
- fix: KeyError when a "primaryKey" is missing ([#1633](https://github.com/frictionlessdata/frictionless-py/issues/1633))
- fix: unexpected field-error for a boolean "example" with "trueValues" or 
  "falseValues" properties ([#1610](https://github.com/frictionlessdata/frictionless-py/issues/1610))

### CODE REVIEW

**Comment by pierrecamilleri** [comment]
@pdelboca @roll 

In the process of releasing the bug fixes in a patch version I've got a "not allowed to push to main error", but the tag has been pushed nonetheless, ending in a somewhat inconsistent state where the Changelog and settings.py are not up to sync on the main branch. 

This PR corrects the situation, but I am not able to merge it by myself (not able to review my own changes). I'd like to take this opportunity to ask you how you intend to proceed with such reviews (do I request one of you as reviewer ?). Thanks !

**Review by pdelboca — APPROVED** [review]

**Comment by pdelboca** [comment]
Hello @pierrecamilleri ! Approved!

For now I guess that just adding @roll  and me as reviewers will work. We should be able to help in a timely manner. 

If we feel we are getting stuck, we can change the rules and allow maintainers to merge without explicit approvals. (A workflow I am a fan off since I rather let other maintainers do and not get blocked, and later review and change if required.)

**Pull request merged**
